db/migrate/20240718183207_add_totp_field_to_user_for_mfa.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddTotpFieldToUserForMfa < Sagittarius::Database::Migration[1.0]
+  def change
+    add_column :users, :totp_secret, :text, unique: { allow_nil_duplicate: true }
```

Comment:
The model already has this validation, so the db should include it as well. You need to re-run this migration, so that `structure.sql` gets updated.
```suggestion
    add_column :users, :totp_secret, :text, unique: { allow_nil_duplicate: true }, limit: 32
```